### PR TITLE
Edit Idea

### PR DESCRIPTION
**Pull request by andorp**
Edit idea page reads value from the database and renders the idea to the user, the user can change it and post back it to the system.

### CODE REVIEW

File: src/Frontend.hs [hunk 1]
```diff
@@ -53,20 +53,22 @@ runFrontend = do
     bootsrapDB persist =
         generate arbitrary >>= void . bootstrapUser persist . (userLogin .~ adminUsernameHack)
 
+type FormHandler = FormH HTML (Html ()) ST
```
Comment (fisx) [inline src/Frontend.hs]: move to `Frontend.Core`?

**Comment by andorp** [comment]
Tests are missing. I am going to refactor the FormPageView in different commit.

**Comment by andorp** [comment]
I added tests.

**Comment by andorp** [comment]
Every idea related functionality points to the `/testings/ideas/...`. I tested the functionality manually.

**Comment by fisx** [comment]
LGTM.

Stiil need to read `formRedirectH'`, and I have some small complaints, but so move things forward I will merge this and we can handle everything in separate PRs.

Thanks!

**Pull request merged**